### PR TITLE
Link to docs/secret.md and kubectl describe in examples/secrets

### PR DESCRIPTION
**Pull request by mwielgus**

### CODE REVIEW

File: examples/secrets/README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Secrets example
 
 Following this example, you will create a secret and a pod that consumes that secret in a volume.
+[Here](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/secrets.md) you can learn
```
Comment (roberthbailey) [inline examples/secrets/README.md]: nit: this would read better as: You can learn more about secrets [here](...).

**Comment by roberthbailey** [comment]
LGTM modulo a bit of wordsmithing.

**Comment by mwielgus** [comment]
@roberthbailey wordsmithing is done.

**Comment by roberthbailey** [comment]
Doc only change. ok-to-merge.

**Pull request merged**